<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-9TCK99PM" failures="0" tests="5" name="com.salesforcenew.testCases.RandonScenariosTests" time="17.692" errors="1" timestamp="2022-02-09T07:01:12 EST" skipped="4">
  <testcase classname="com.salesforcenew.testCases.RandonScenariosTests" name="blockingEventWeeklyRecurrance37" time="17.692">
    <error message="class jdk.proxy2.$Proxy13 cannot be cast to class java.util.List (jdk.proxy2.$Proxy13 is in module jdk.proxy2 of loader &#039;app&#039;; java.util.List is in module java.base of loader &#039;bootstrap&#039;)" type="java.lang.ClassCastException">
      <![CDATA[java.lang.ClassCastException: class jdk.proxy2.$Proxy13 cannot be cast to class java.util.List (jdk.proxy2.$Proxy13 is in module jdk.proxy2 of loader 'app'; java.util.List is in module java.base of loader 'bootstrap')
at com.salesforcenew.testCases.RandonScenariosTests.blockingEventWeeklyRecurrance37(RandonScenariosTests.java:232)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- blockingEventWeeklyRecurrance37 -->
  <system-out/>
  <testcase classname="com.salesforcenew.testCases.RandonScenariosTests" name="verifyEditedLastName34">
    <skipped/>
  </testcase> <!-- verifyEditedLastName34 -->
  <system-out/>
  <testcase classname="com.salesforcenew.testCases.RandonScenariosTests" name="verifyTheTabCustoization35">
    <skipped/>
  </testcase> <!-- verifyTheTabCustoization35 -->
  <system-out/>
  <testcase classname="com.salesforcenew.testCases.RandonScenariosTests" name="blockingAnEvent36">
    <skipped/>
  </testcase> <!-- blockingAnEvent36 -->
  <system-out/>
  <testcase classname="com.salesforcenew.testCases.RandonScenariosTests" name="verifyFirstNameLastName33">
    <skipped/>
  </testcase> <!-- verifyFirstNameLastName33 -->
  <system-out/>
</testsuite> <!-- com.salesforcenew.testCases.RandonScenariosTests -->
